### ISSUE TITLE
Error in line 23

### ISSUE THREAD

**Issue by MarkusKonk** [alfcrisci/EMS_codified#1]
Hi @alfcrisci, hi @valenitna, 
I tried to rerun the script and encountered the following in error in line 22: codified_df_native=... 
`Error in [.data.frame(x$dmeta, tag) : undefined columns selected` 

Do you have an idea how to solve it?

Thanks in advance,

Markus

PS: I am referring to the fork by @valenitna
PPS: I think it is because of the parameter `corpus_analysis=TRUE`. The error is gone if I set it to FALSE but this is not wanted I guess.

**Comment by alfcrisci** [comment]
Hi @MarkusKonk i reframed my package rTwChannel.

https://github.com/alfcrisci/rTwChannel/blob/master/R/main_data_channel.r
reinstall the one by using 

devtools::install_github("alfcrisci/rTwChannel")

Please consider the code in the repository  deprecated.
Tell me if the example goes well.
Alfonso

**Comment by MarkusKonk** [comment]
Hi @alfcrisci 

Thank you for taking a look at it. But I am still running into the same error.

**Comment by alfcrisci** [comment]
Thanks for your interest  @MarkusKonk  give me a couple of days and i return to you a full reproducible code. After we update the github page of @valenitna 's repo.
The problem seems on the native's twitter channel.
I check all and i will inform you on this issue.
alf

**Comment by alfcrisci** [comment]
Hi  @MarkusKonk i 've patched my package. The issue comes from a text-mining packages to import and share corpora data. Reinstall and upgrade qdap, tm and rTWChannel  and try again.
I have update 
Soon

**Comment by MarkusKonk** [comment]
@alfcrisci Mhh I installed and included qdap, tm and rTwChannel by using devtools::install_github(...) but still have a similar issue which looks slightly different: `Error in [.data.frame(word_freq_matr, c("word", "grouping.var")) :   undefined columns selected`

**Comment by alfcrisci** [comment]
i try to check again @MarkusKonk.

**Comment by alfcrisci** [comment]
yes @MarkusKonk sorry , i have made further correction in  my code. Please re install rTwChannel package.

**Comment by MarkusKonk** [comment]
@alfcrisci 
Sorry for the late response. I was not in my office the last weeks. 
The problem above is solved. Great! 
But I ran into another issues in line 75 `res=main_data_channel(codified_native_LIG,name="LIG_native",native=T,corpus=T,filterdegree = 5)`
which gives the following error: `Error in data.frame(name = node_names, group = 1, size = 8) : arguments imply differing number of rows: 0, 1 `
It works if I leave out filterdegree=5 but I am not sure about the effects on the results. It is a recurring error in the lines 99 and 122.
Do you have an idea?
